richtextfx/src/integrationTest/java/org/fxmisc/richtext/style/StylingTests.java

```diff
@@ -0,0 +1,160 @@
+package org.fxmisc.richtext.style;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.fxmisc.flowless.Cell;
+import org.fxmisc.flowless.VirtualFlow;
+import org.fxmisc.richtext.InlineCssTextAreaAppTest;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.nitorcreations.junit.runners.NestedRunner;
+
+import javafx.scene.Node;
+import javafx.scene.layout.Region;
+import javafx.scene.shape.Path;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextFlow;
+
+@RunWith(NestedRunner.class)
```

Comment:
NestedRunner is only necessary if you want to use a class to set up common things among multiple tests. Since you're not doing that here, you can remove it.

Reply:
True. It was essentially a preparation for later enhancements, but it is indeed better to add it once it is really required.

Reply:
I like NestedRunner because it reduces code duplication when setting up similar tests. However, it doesn't allow one to run tests within the nested classes in my IDE. I'm not sure how to fix that, but even if we did, I would think that such a thing should not need to be explained for the tests to work on someone else's computer.